aspnet/2-structured-data/Models/DbBookStore.cs

```diff
@@ -46,12 +46,13 @@ public void Delete(long id)
         // [START list]
         public BookList List(int pageSize, string nextPageToken)
         {
-            var pageOfBooks = (null == nextPageToken ?
-                (from book in _dbcontext.Books orderby book.Id select book) :
-                (from book in _dbcontext.Books
-                 where book.Id > Int32.Parse(nextPageToken)
-                 orderby book.Id
-                 select book)).Take(pageSize + 1);
+            IQueryable<Book> query = from book in _dbcontext.Books orderby book.Id select book;

```

Comment:
Hmm.  Since `System.Linq` `IEnumerable` extension methods, I've favored lambda expressions over query syntax.  I find it to be more concise and readable.  I wonder if one is more idiomatic than the other for these use-cases?  For this, for example, I find it *really strange* that you have to specify that you want to `select book` `from books` when you're *already* querying the `Books` table.

That LOC repeats the fact that you want to query a `Book` 3 separate times? :flushed:

Either way, the query syntax is 93 characters (vs 73).  Might want to consider adding newlines.